docs/infrastructure/administration.md

```diff
@@ -0,0 +1,416 @@
+# Infrastructure administration
+
+## Obtaining access
+
+- Heroku: Follow the [Mozilla Heroku SSO guide] then ask to be invited to the Heroku apps.
+- Amazon RDS: File a [Treeherder infrastructure bug] requesting access to the
+  `moz-devservices` AWS account and needinfo `:dividehex`, who will update the
+  [IAM configuration file][iam-config].
+- New Relic: Ask for an invite to be sent using the [New Relic user management] page.
+- Papertrail: Ask for an invite to be sent using [Papertrail user management] page.
```

Comment:
Ask who? Someone on the IRC channel?

Reply:
Anyone who has the user-management manage permission on the Papertrail account can give access.

Reply:
I've updated this entry